indico/modules/users/export_schemas.py

```diff
@@ -161,12 +172,13 @@ class Meta:
         model = Registration
         fields = ('id', 'url', 'event_id', 'event_title', 'event_url', 'state', 'currency', 'price', 'submitted_dt',
                   'email', 'first_name', 'last_name', 'is_deleted', 'checked_in', 'checked_in_dt', 'rejection_reason',
-                  'consent_to_publish', 'created_by_manager', 'fields')
+                  'consent_to_publish', 'created_by_manager', 'fields', 'documents')
 
     price = fields.Decimal(as_string=True)
     url = fields.Function(lambda reg: url_for('event_registration.registration_details', reg, _external=True))
     event_title = fields.Function(lambda reg: reg.event.title)
     event_url = fields.Function(lambda reg: url_for('events.display', reg.event, _external=True))
+    documents = fields.List(fields.Nested(ReceiptFileExportSchema))
```

Comment:
You'll need to manually specify the attribute name since the model calls it `receipt_files`

```suggestion
    documents = fields.List(fields.Nested(ReceiptFileExportSchema), attribute='receipt_files')
```